packages/commonwealth/client/scripts/views/components/topic_selector.tsx

```diff
@@ -44,6 +44,7 @@ export const TopicSelector = ({
       options={allOptions}
       className="TopicSelector"
       onChange={handleOnChange}
+      value={topicToOption(value, false)}
```

Comment:
I assumed uses just `value` would work here but causes multiple down-line type issues. To the best of my knowledge this works with zero regressions, would be interested in if there's another obvious solution... taking the time crunch to launch gating in consideration. 

cc: @rbennettcw @mzparacha